NextcloudTalk/CalendarEvent.swift

```diff
@@ -0,0 +1,29 @@
+//
+// SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+// SPDX-License-Identifier: GPL-3.0-or-later
+//
+
+import Foundation
+
+@objcMembers public class CalendarEvent: NSObject {
```

Comment:
```suggestion
public class CalendarEvent {
```

If only used from swift, we can remove the objc interoperability (need to mark the api method with `@nonobjc`). In theory can also be a  `struct` then.